gossip/gossip.go

```diff
@@ -352,6 +352,7 @@ func (g *Gossip) bootstrap() {
 		g.mu.Lock()
 		g.parseBootstrapAddresses()
 		if g.closed {
+			g.mu.Unlock()
```

Comment:
But this is using two different objects (`g.mu` and `g.stalled`). If anything, the lines below should be

```
g.stalled.L.Lock()
g.stalled.Wait()
g.stalled.L.Unlock()
```

or am I missing something?